### PR TITLE
Auditissue

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review it has new dashboard layout and some fixes

### CODE REVIEW

**Comment by Devilla** [comment]
Merge the changes to development.

File: src/components/Header/ConnectionStatus.scss [hunk 1]
```diff
@@ -40,9 +40,11 @@ div #status-noteConnected {
   #status-noteConnected {
     top: 0;
     border-radius: 0 0 6px 6px;
-    padding: 5px 75px;
-    right: 50%;
-    background: #097fff;
+    padding: 5px 20px;
+    right: 45%;
+    /* Change background color to brand color */
+    /* background: #097fff; */
```
Comment (ShankyRana) [inline src/components/Header/ConnectionStatus.scss]: Remove

File: src/components/Header/ConnectionStatus.scss [hunk 2]
```diff
@@ -91,8 +93,10 @@ div #status-noteConnected {
   
   .center-block {
     /* display: block; */
```
Comment (ShankyRana) [inline src/components/Header/ConnectionStatus.scss]: Remove

File: .env [hunk 3]
```diff
@@ -1,5 +1,5 @@
-NODE_PATH=src/
+  NODE_PATH=src/
 REACT_APP_PRODUCTION_URL="https://strapi.useinfluence.co/"
-REACT_APP_STAGING_URL="https://api.useinfluence.co/"
-REACT_APP_DEVELOPMENT_URL="https://strapi.useinfluence.co/"
+REACT_APP_STAGING_URL="http://localhost:1337/"
```
Comment (ShankyRana) [inline .env]: revert these to original

File: src/components/Integration/Integration.js [hunk 4]
```diff
@@ -1,20 +1,141 @@
...
     super();
+    this.state = {
+      selectedChannels: [],
+      channelContent: [],
+      checked: false
+    };
+  }
+
+  componentWillMount() {
+    window.scrollTo(0,0);
```
Comment (ShankyRana) [inline src/components/Integration/Integration.js]: change this.

File: src/components/Oauthpage/Oauthpage.js [hunk 5]
```diff
@@ -9,10 +10,10 @@ class Oauthpage extends Component {
   constructor() {
```
Comment (ShankyRana) [inline src/components/Oauthpage/Oauthpage.js]: remove empty contructor, if not needed.

File: src/components/WebsiteFooter/WebsiteFooter.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
+.websitefooter-container {}
```
Comment (ShankyRana) [inline src/components/WebsiteFooter/WebsiteFooter.scss]: remove

Comment (GAUTAMRAJU15) [reply src/components/WebsiteFooter/WebsiteFooter.scss]: done

File: src/components/Oauthpage/Oauthpage.js (hunk 5 above)
Comment (GAUTAMRAJU15) [reply src/components/Oauthpage/Oauthpage.js]: done

File: src/components/Integration/Integration.js (hunk 4 above)
Comment (GAUTAMRAJU15) [reply src/components/Integration/Integration.js]: done

File: .env (hunk 3 above)
Comment (GAUTAMRAJU15) [reply .env]: done

**Pull request closed without merging**